DOCUMENTATION.md

```diff
@@ -284,9 +291,73 @@ Revoke support access to an application.
 #### Examples:
     >> > resin.models.application.revoke_support_access('5685')
     'OK'
+## Build
+
+This class implements build model for Resin Python SDK.
+### Function: get(id)
+
+Get a specific build.
+
+#### Args:
+    id (str): build id.
+
+#### Returns:
+    dict: build info.
+
+#### Raises:
+    BuildNotFound: if build couldn't be found.
+### Function: get_all_by_application(app_id, include_logs)
+
+Get a specific build.
+
+#### Args:
+    app_id (str): application id.
+    include_logs (Optional[bool]): True if user wants to include build logs in build info, False otherwise since these may be very large.
```

Comment:
IMHO it'd still be useful for the docs to mention what this optional parameter defaults to.

Reply:
@lurch sorry, just forgot to update.